src/common/registry/test/validator-registry/registry-update.spec.ts

```diff
@@ -339,3 +324,90 @@ describe('Empty registry', () => {
     await registryService.update(address, blockHash);
   });
 });
+
+describe('Reorg detection', () => {
+  const provider = new JsonRpcBatchProvider(process.env.PROVIDERS_URLS);
+  let registryService: ValidatorRegistryService;
+  let registryStorageService: RegistryStorageService;
+  let moduleRef: TestingModule;
+  const mockCall = jest.spyOn(provider, 'call').mockImplementation(async () => '');
+  const CHAIN_ID = process.env.CHAIN_ID || 1;
```

Comment:
i think some tests have fixtures for chainId = 5 , and if someone forget to set CHAIN_ID in env file will got some strange errors

Reply:
I propose to do it as part of another task